src/components/Form.js

```diff
@@ -0,0 +1,23 @@
+// import { useState } from "react";
+// import ErrorMessage from "./ErrorMessage";
+
+const Form = (setChosenDate) => {
```

Comment:
Once the setChosenDate function is passed as a prop, please deconstruct the function using curly brackets:
```js
const Form = ({setChosenDate}) => { ...
```